beacon_node/src/cli.rs

```diff
@@ -835,4 +835,11 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                        failure caused by the execution layer.")
                 .takes_value(false)
         )
+        .arg(
+            Arg::with_name("disable-eth1-sync")
```

Comment:
Since the merge we're trying to avoid the "eth1" terminology, I think we could name this flag something related to the deposit contract, as that's how most of our logs now refer to it.

```suggestion
            Arg::with_name("disable-deposit-contract-sync")
```

Reply:
Done. The description still mentions `eth1` to remain consistent with the other flags. Should I change it too?

Reply:
Yeah I think that would be good. Eventually we should phase out the `--eth1-endpoints` and `--eth1` flags entirely (maybe after the Gnosis merge).

Reply:
[better ?](https://github.com/sigp/lighthouse/pull/3597/commits/5c212039af049f3fa7cd34b03e743b54fa7793d2)

Reply:
yep!